plugin-server/src/worker/ingestion/persons/repositories/postgres-person-repository.ts

```diff
@@ -213,6 +423,25 @@ export class PostgresPersonRepository
                 }
             }
 
+            if (this.isPropertiesSizeConstraintViolation(error)) {
+                // For createPerson, we just log and reject since there's no existing person to update
+                personPropertiesSizeViolationCounter.inc({
+                    violation_type: 'create_person_size_violation',
+                })
+
+                logger.warn('Rejecting person properties create/update, exceeds size limit', {
+                    team_id: teamId,
+                    person_id: undefined,
+                    violation_type: 'create_person_size_violation',
+                })
+
+                throw new PersonPropertiesSizeViolationError(
+                    `Person properties create would exceed size limit`,
+                    teamId,
+                    undefined
+                )
```

Comment:
question: Are we failing gracefully in this case? I could not find the code that handles the size violation exceptions when creating a person, but I might be looking wrong. I think it would be good to test it at the `person-*-service` level.